cluster-bootstrap/README.md

```diff
@@ -6,27 +6,51 @@ This folder contains the artifacts with configuration and related Argocd applica
 
 
 ### Prerequisites:
-Make sure all secrets(path and key) are put in Vault before the deployment. Please refer to the secret definition doc here:
+Please check the following doc which contains the tokens need to use during the cluster bootstrap deployment.
 https://docs.google.com/document/d/1E5n62ed9-ls3rIIPqd8SoTM2W9OzC6xQTq6jQc11fsA/edit
 
 
 ### Deployment:
-1. Replace the `VAULT_ADDRESS` and `VAULT_TOKEN` in `cluster-bootstrap/openshift-gitops/config/argocd.yaml` with the Vault service address and auth token which are available in above doc.
-2. Run `make deploy-dev-all`.
+1. Replace the `VAULT_ADDRESS` and `VAULT_TOKEN` in `cluster-bootstrap/openshift-gitops/config/argocd.yaml` with the Vault service address and auth token from above doc.
+   Please note that to used the correct env values for your deployment. 
+   * Dev is for demo and development env usage. 
```

Comment:
make sure dev valut conf only use for the dev env. need to have another valut path for demo or local env.

Reply:
Ok. Update the doc